patch/cisco-x86-gpio-config.patch

```diff
@@ -0,0 +1,42 @@
+From 2ee30f75a96d5f7841ffd58cd65410160302421b Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Tue, 13 Jul 2021 18:43:16 -0700
+Subject: [PATCH] Configurable GPIO option ARCH_NR_GPIO for x86 architecture
```

Comment:
[Please make it a statement.](https://chris.beams.io/posts/git-commit/)

Reply:
Updated patch description following example from kernel.org for similar change